charts/retool/templates/_helpers.tpl

```diff
@@ -230,6 +284,20 @@ Set Temporal namespace
 {{- end -}}
 {{- end -}}
 
+{{/*
+Set workflow backend service name
+*/}}
+{{- define "retool.workflowBackend.name" -}}
+{{ template "retool.fullname" . }}-workflow-backend
+{{- end -}}
+
+{{/*
+Set workflow worker service name
+*/}}
+{{- define "retool.workflowWorker.name" -}}
+{{ template "retool.fullname" . }}-workflow-worker
+{{- end -}}
```

Comment:
These result in the exact same names as before, just a template code refactor for reuse.